### DISCUSSION TITLE
Remarks to the Wiki-Section > Configure

### DISCUSSION THREAD

**Discussion by versor2k7** [MiczFlor/RPi-Jukebox-RFID#2215] [General]
Hello, I have noticed some things in the wiki area:

1. [Web app](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/CONFIGURE#running-the-web-app): Everything described in this section already worked for me after the installation. That was a good thing, because the /etc/php/7.0/fpm/ folder no longer exists with the procedure currently described under "Install".
2. [Autostart Phonie-Box](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/CONFIGURE#auto-start-the-phoniebox): You refer to the old button script when starting the service:
`sudo systemctl start phoniebox-gpio-buttons`
It should be:
`sudo systemctl start phoniebox-gpio-control`

Thanks for your work. Versor

**Comment by AlvinSchiller** [comment]
Hi, thanks for pointing out :)
The information one the [CONFIGURE](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/CONFIGURE) page should all be integerated in the installation process by now, so no need to go over it again.
We should state that more clearly and update / remove some outdated docs.

I will have a look at it 👍
